Description: add the actual binary names from the qemu packages so default
 hypervisor detection works properly on first run.
Author: Marc Deslauriers <marc.deslauriers@canonical.com>
Updated: 2015-08-22

Index: virt-manager-1.4.2/virtManager/connect.py
===================================================================
--- virt-manager-1.4.2.orig/virtManager/connect.py	2017-08-09 17:51:42.377171919 +0200
+++ virt-manager-1.4.2/virtManager/connect.py	2017-08-09 17:51:42.373171976 +0200
@@ -117,6 +117,8 @@
         if (os.path.exists("/usr/bin/qemu") or
             os.path.exists("/usr/bin/qemu-kvm") or
             os.path.exists("/usr/bin/kvm") or
+            os.path.exists("/usr/bin/qemu-system-x86_64") or
+            os.path.exists("/usr/bin/qemu-system-i386") or
             os.path.exists("/usr/libexec/qemu-kvm") or
             glob.glob("/usr/bin/qemu-system-*")):
             return "qemu:///system"
